src/request.rs

```diff
@@ -47,8 +47,14 @@ use futures::Future;
 mod ask;
 mod tell;
 
-pub use ask::{AskRequest, LocalAskRequest, RemoteAskRequest};
-pub use tell::{LocalTellRequest, RemoteTellRequest, TellRequest};
+#[cfg(feature = "remote")]
+pub use ask::RemoteAskRequest;
+
+#[cfg(feature = "remote")]
+pub use tell::RemoteTellRequest;
```

Comment:
These types don't get marked as feature gated due to the missing annotations.

Reply:
Edit: if we go with the `doc_auto_cfg` suggestion, then we won't need this anymore